SparkPost/2web2ui
# FE-1412 - Fixes misspelled words on the domain details pages

[FE-1412](https://sparkpost.atlassian.net/browse/FE-1412)

### What Changed

- Fixes typos
- Wraps content in a missing `<p>` tag via the `Text` component

### How To Test

- Go to `/domains/details/sending-bounce/<some-domain>`
- Verify the typos are fixed
- Verify that a `<p>` element wraps the content

### To Do
